bitnami/kafka/Chart.yaml

```
@@ -24,4 +24,4 @@ name: kafka
 sources:
   - https://github.com/bitnami/bitnami-docker-kafka
   - https://kafka.apache.org/
-version: 12.0.0
+version: 12.0.1
```

Hi @thebigcorporation,

Thank you for the PR! As it is a minor fix on that only changes the README and not the code, there is no need to bump the chart version.

```suggestion
version: 12.0.0
```

---

Glad it got addressed; the version bump was apparently required by CI, perhaps there is a whitelist that can be applied for documentation changes.